### DISCUSSION TITLE
Helpers: remove

### DISCUSSION THREAD

**Discussion by alfonsor** [ujjwalguptaofficial/JsStore#349] [Q&A] [answered]
I congratulate you for this magnificent project. I am implementing your solution and I have a doubt: In the [Helpers](https://jsstore.net/docs/helpers), we have set and get, but there is no remove. What do I do when we no longer need a key?

Thank you!

**Comment by ujjwalguptaofficial** [comment]
Yeah this has not been added, I will add this later. The plan is convert this into equivalent of javascript `Map` . I will work on this and let you know.

**Comment by ujjwalguptaofficial** [comment]
@alfonsor Thank you for your kind words - It means a lot :).

**Comment by ujjwalguptaofficial** [comment, marked as answer]
@alfonsor implemented the map api in v - 4.7.0.

now similar to javascript `Map`, you have `get`, `set`, `delete` and `has` API.

```
connection.Map.get('key')
```

I will update the docs in some time.

**Reply by ujjwalguptaofficial** [reply]
here is doc for map - https://jsstore.net/docs/map

**Reply by alfonsor** [reply]
Your response time is incredible! From request to fully documented implementation **in just 15 hours**!

Looks great. I'll install it and get back to you.

Thank you very much!

**Comment by alfonsor** [comment]
@ujjwalguptaofficial I have already tested it and it works very well!

Thank you very much for all the effort.
